src/App.js

```diff
@@ -91,6 +77,25 @@ class App extends Component {
       </React.Fragment>
```

Comment:
Fun fact: you can create a react fragment with the shorthand <> </>

Reply:
that is a fun fact!